test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -34,12 +34,31 @@ using namespace dev;
 using namespace dev::eth;
 namespace fs = boost::filesystem;
 
+namespace {
```

Comment:
Brace should be on a separate line. Better use clang-format. Also convert tabs to spaces (AFTER clang-formatting, maybe better in separate commit) in the files you touch

Reply:
I will run clang-format on all touched files in a separate commit.